packages/shared/notionable-hooks/src/use-wallet.ts

```diff
@@ -128,16 +128,22 @@ export function useWalletBalances(
         token,
         content: { 
           balance: maxBalance?.isPositive() ? maxBalance?.toDisplayStringWithSymbol(3, true) : undefined, 
+          usdBalance: maxBalance?.isPositive() ? maxBalance?.toFiat('USD').toDisplayString() : undefined,
```

Comment:
You can use `toFloat()` here instead of `toDisplayString()` and then you won't have to `parseFloat` later